testing/path/test_svnwc.py

```diff
@@ -113,8 +112,9 @@ def test_status_unchanged(self, path1):
         assert r.join('sampledir/otherfile').basename in [item.basename
                                                     for item in s.unchanged]
 
-    @pytest.mark.xfail(reason="svn-1.7 has buggy 'status --xml' output")
     def test_status_update(self, path1):
+        # not a mark because the global "pytestmark" will end up overwriting a mark here
+        pytest.xfail("svn-1.7 has buggy 'status --xml' output")
```

Comment:
this is fixed in recent pytest and should be in the next feature release (the evaluator refactoring)